### ISSUE TITLE
Line number

### ISSUE THREAD

**Issue by poggio84** [zatonovo/futile.logger#61]
Hi, is it possible to print in the log also the line number?

Thanks
Diego

**Comment by muxspace** [comment]
Hi, do you want to print the line number of the log message? That might be a bit tricky as it requires parsing the source code. Not sure how efficient that would be on the fly.

**Comment by poggio84** [comment]
Yes.. like in log4j where you can use the L option to set the line number from where the logging request was issued.
See:
https://www.mkyong.com/logging/log4j-hello-world-example/
https://logging.apache.org/log4j/1.2/apidocs/org/apache/log4j/PatternLayout.html
http://alvinalexander.com/blog/post/java/sample-how-format-log4j-logging-logfile-output
